### PR TITLE
Quarto theme v2updates

### PR DESCRIPTION
**Pull request by AshleyHenry15**
### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

Tracking issue is here: https://github.com/rstudio/rstudio-pro/issues/6198

I updated the extension to v2 which updates the fonts, fixes spacing in the toc and sidebar, and also simplifies implementation moving forward.

Testing information and changes is in the issue. 

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

- News
- User RStudio

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [ ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

### CODE REVIEW

File: version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md [hunk 1]
```diff
@@ -36,6 +36,8 @@
 - RStudio now prints a warning if an overly-long line of console input is truncated when sending to R (#14420)
 - Updated zlib to 1.3.1 (rstudio/rstudio-pro#5971)
 - Vim: Fixed an issue where bracket highlight followed the bracket behind the cursor when in normal mode (#4152)
+- Updated nginx to 1.25.4 (rstudio/rstudio-pro#5970)
```
Comment (melissa-barca) [inline version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:39]: Git is picking up some changes to this file that are unrelated to your changes. I think if you pull main into your branch it should take care of them?

Comment (AshleyHenry15) [reply version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:39]: Pull `main` and not `rel-chocolate-cosmos`?

Comment (AshleyHenry15) [reply version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md]: Ah - it looks like although my local branch was against `rel-chocolate-cosmos` - I failed to change the comparison branch on the PR here. Gets me everytime...

**Review by melissa-barca — APPROVED** [review]
LGTM

**Review by MariaSemple — APPROVED** [review]
LGTM!
